google-map-marker.html

```diff
@@ -192,7 +195,8 @@
       longitude: {
         type: Number,
         value: null,
-        reflectToAttribute: true
+        reflectToAttribute: true,
```

Comment:
@ebidel should I remove the `reflectToAttribute: true` for latitude and longitude properties?